lib/src/models/multireddit.dart

```diff
@@ -264,13 +282,18 @@ class Multireddit extends RedditBase with RedditBaseInitializedMixin {
   /// [subreddit] is the name of the [Subreddit] to be added to this [Multireddit].
   Future add(/* String, Subreddit */ subreddit) async {
     final subredditName = _subredditNameHelper(subreddit);
+    final newSubredditObject = {'name': "$subredditName"};
+    final encoder = new JsonEncoder.withIndent('  ');
```

Comment:
This is probably a bit overkill for a single object.

Reply:
ok